### ISSUE TITLE
Sub Switching Issue

### ISSUE THREAD

**Issue by perryhuynh** [mrmin123/kancolle-auto#288]
*Please read the FAQ/Common Issues section of the README before opening a ticket. Please remove sections of this template you do not find relevant when creating your ticket.*

#### Environment details

* kancolle-auto version (date on first line of your CHANGELOG.md): `2017-02-14`
* Viewer/browser: `Chromium + KC3`
* Operating system: `Xubuntu 16.04`

#### Issue
Is the issue consistent (can be reproduced consistently) or is it transient (only happens sometimes)?:

Consistent. 

Submarine switching mistakes out `i-26` for `i-58`.

`u-511` was sent for repair, then the script selected `i-26` and replaced with `i-58` even though `i-26` was not under repair.

Notes: I don't know if `i-58` specifically is the cause or that `i-58` just happens to be on the same page as `i-26`, but that doesn't explain why `i-26` was chosen for replacement even though she was not damaged.

#### Console log (if applicable)
```
[replace this line with ~50 lines of console log]
```

#### Relevant config snippet (if applicable)
```
EnabledSubs: i-26, u-511
```

**Comment by mrmin123** [comment]
Might be due to the image match being 'just close enough' as far as Sikuli is concerned... needs furthering testing to generate a better % match threshold.

**Comment by mrmin123** [comment]
Merging in and continuing work in #303

**Comment by mrmin123** [comment]
Closed with #303.

Should be fixed now @perryhuynh
